doc/modules/multi_column_operations/advanced_selectors.rst

```diff
@@ -103,3 +103,35 @@ do this like this:
 2  History -1.069045
 3  Science  0.267261
 4      Art -1.069045
+
+
+Using :func:`filter` to select columns that contain outliers
+------------------------------------------------------------
+The :func:`filter` selector can be used to select columns based on custom
+criteria. For example, we can define a function that checks if a column contains
+outliers using the Interquartile Range (IQR) method, and then use this function
+with :func:`filter` to select such columns.
+
+>>> def has_outliers(column):
+...    q1 = column.quantile(0.25)
+...    q3 = column.quantile(0.75)
+...    IQR = q3 - q1
+...    lower_bound = q1 - 1.5 * IQR
+...    upper_bound = q3 + 1.5 * IQR
```

Comment:
```suggestion
...    lower_bound = q1 - 2 * IQR
...    upper_bound = q3 + 2 * IQR
```

I find 1.5 a bit of a low range extension. Changing it to 2 (the example still works)